procedures: Add PostgreSQL migration guides (#2163)
codenvy/che-docs: modules/administration-guide/partials/assembly_migration-from-helm-charts-to-operator.adoc

:parent-context-of-migration-from-helm-charts-to-operator: {context}

[id="migration-from-helm-charts-to-operator_{context}"]
= Migration from Helm Charts to Operator

:context: migration-from-helm-charts-to-operator

> Follow the procedures below to migrate from Helm Charts to Operator successfully without any data loss.

Reviewer: ```suggestion
You can migrate from Helm Charts to Operator without losing data.
```
